working-open.md

```diff
@@ -0,0 +1,53 @@
+# Working Open
+
+Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that, we've developed a set of guidelines for how (and exactly what) and why we work in the open.
+
+## Why?
+
+- In order to share and provide opportunities for self- and collective- learning as we work in solidarity with others seeking to create meaningful livelihoods.
+- Not because "default to open" is an ideal in itself, but because it orients us toward negotiating a set of practices around our shared intentions.
```

Comment:
```suggestion
- Not because "default to open" is an ideal in itself, but because it orients us toward negotiating a set of practices around our shared intentions
```

Reply:
You'd like to remove the periods?

Reply:
Reviewed and we err on the side of periods at the end of full sentences, including bullet points, both in this doc and in the other committed material. I'd rather add the one missing period than remove all of these

Reply:
I ever-so-slightly prefer punctuation on bullets too :) Esp with markdown. If you agree, perhaps mark these as resolved @benhylau?

Reply:
@dcwalk I have no preference other than being consistent.

Reply:
Okay, resolving these suggestions then